x230/README.md

```diff
@@ -18,32 +18,29 @@ Get it from our [release page](https://github.com/merge/coreboot-x230/releases)
 * [How to rebuild](#how-to-reproduce-the-release-images)
 
 ## TL;DR
-1. run `sudo ./x230_before_first_install.sh` on your current X230 Linux system
-2. Power down, remove the battery. Remove the keyboard and palmrest. Connect
-a hardware flasher to an external PC (or a Raspberry Pi with a SPI 8-pin chip clip
-can directly be used), and run
-`sudo ./external_install_bottom.sh` on the lower chip
-and `sudo ./external_install_top.sh` on the top chip of the two.
-3. For updating later, run `./x230_skulls.sh`. No need to disassemble.
+1. If your Thinkpad is already running linux: run `sudo ./x230_before_first_install.sh` on it
+2. Power down, remove the battery. Remove the keyboard and palmrest.
+3. Connect a hardware flasher to an external PC (or a Raspberry Pi with a SPI 8-pin chip clip
+can directly be used)
+4. Run `sudo ./external_install_bottom.sh` on the lower chip
+5. Run `sudo ./external_install_top.sh` on the top chip of the two
+6. Optionally: For updating later, run `./x230_skulls.sh`. No need to disassemble.
 
 And always use the latest [released](https://github.com/merge/coreboot-x230/releases)
-package. This will be tested. The git master
-branch is _not_ meant to be stable. Use it for testing only.
+package. This will be tested. The git master branch is _not_ meant to be stable. Use it for testing only.
 
 ## First-time installation
-#### before you begin
-Before starting, run Linux on your X230, install `dmidecode` and run
-`sudo ./x230_before_first_install.sh`. It simply prints system information and
-helps you to be up to date.
-Also make sure you have the latest skulls-x230 package release by running `./upgrade.sh`.
 
-#### original BIOS update / EC firmware (optional)
+### If you are still on Windows and Lenovo BIOS
+
 Before flashing coreboot, consider doing one original Lenovo upgrade process
 in case you're not running the latest version. This is not supported anymore,
 once you're running coreboot (You'd have to manually flash back your backup
 images first, see later chapters).
 
-Also, this updates the BIOS _and_ Embedded Controller (EC) firmware. The EC
+Check the [Lenovo Support site (which is quite good and actually helpful)](https://pcsupport.lenovo.com/de/en/products/laptops-and-netbooks/thinkpad-x-series-laptops/thinkpad-x230) and e.g. run the _Lenovo System Update for Windows_ to check for old BIOS, EC- or battery-firmware or other updateable firmwar.
+
+Also, this updates the BIOS (latest 2.74) _and_ Embedded Controller (EC) firmware. The EC
```

Comment:
no, sorry. first, we don't differentiate between running windows or linux here. Lenovo provides bootable ISO. Also, we don't want to maintain the latest BIOS version number in the README too.

Reply:
It would make sense to differentiate between Linux/Windows because coming from Windows it is not possible to run "x230_before_first_install.sh" without further preparation.
Just added the BIOS version because EC version was in the text. Not important.